### ISSUE TITLE
Support `therun.gg`

### ISSUE THREAD

**Issue by CryZe** [LiveSplit/livesplit-core#593]
Now that https://therun.gg is gaining traction we want to support uploading entire runs and synchronize in-progress runs. The latter is likely blocked by #260 (at least automatically doing so, we can still provide the functionality).

**Comment by TheTaraStark** [comment]
Is this closed?

**Comment by wooferzfg** [comment]
I don't think this is done yet

**Comment by CryZe** [comment]
Yeah Tara thought this repository also relates to the original LiveSplit.

**Comment by CryZe** [comment]
Fixed by #903
